packages/nodes-from-anchor/test/v01/typeNodes/TupleTypeNode.test.ts

```diff
@@ -0,0 +1,13 @@
+import { numberTypeNode, publicKeyTypeNode, tupleTypeNode } from '@kinobi-so/nodes';
+import test from 'ava';
+
+import { typeNodeFromAnchorV01 } from '../../../src/index.js';
+
+test('it creates tuple type nodes', t => {
```

Comment:
We should probably add a test here that makes sure we throw an error when we end up with a Anchor IDL struct with a mixed of named and unnamed fields.